cervicalcancer/src/main/assets/forms/treatment_cryo_reproductive_health.json

```diff
@@ -56,11 +56,11 @@
           ]
         },
         {
-          "tag": "last menstrual period",
+          "tag": "last_menstrual_period",
           "conceptId": 165202,
-          "widgetType": "DatePickerButton",
+          "widgetType": "DatePicker",
           "weight": 1,
-          "text": "Date of Last Menstrual Period ",
+          "hint": "Date of Last Menstrual Period ",
```

Comment:
@csamiselo 
Add text describing purpose of the date picker in the `label` attribute support for that has been added on the commit:
https://github.com/BlueCodeSystems/smartcerv/pull/58/commits/18acc79e6cab244f75cbde1eb4b2ca71584cecc0 
Includes an example of how to declare the widget data.

Reply:
noted